alerts/apps_alerts.libsonnet

```diff
@@ -25,7 +25,13 @@
           },
           {
             expr: |||
-              sum by (namespace, pod) (max by(namespace, pod) (kube_pod_status_phase{%(prefixedNamespaceSelector)s%(kubeStateMetricsSelector)s, phase=~"Pending|Unknown"}) * on(namespace, pod) group_left(owner_kind) max by(namespace, pod, owner_kind) (kube_pod_owner{owner_kind!="Job"})) > 0
+              sum by (namespace, pod) (
+                max by(namespace, pod) (
+                  kube_pod_status_phase{%(prefixedNamespaceSelector)s%(kubeStateMetricsSelector)s, phase=~"Pending|Unknown"}
+                ) * on(namespace, pod) group_left(owner_kind) topk by(namespace, pod) (
```

Comment:
Can we add some comments for these? I think I can tell from reading but it would be nice to add why we’re doing topk(1, ...)

Reply:
fair enough :)